server/views/pages/edit/singleField.njk

```diff
@@ -0,0 +1,64 @@
+{% extends "../index.njk" %}
```

Comment:
Is this in use anywhere?

Reply:
Not at the moment - it was used by the height page with the intent to be used by the weight page until I got the designs and needed the switcher, figured I'd keep it in for now at least 😅